packages/react/scroll-area/src/ScrollArea.tsx

```diff
@@ -50,6 +50,14 @@ const SCROLL_AREA_CSS_PROPS_LIST = [
   'cornerHeight',
 ] as const;
 const SCROLL_AREA_CSS_PROPS = SCROLL_AREA_CSS_PROPS_LIST.reduce(reduceToCssProperties, {} as any);
+const AUTO = 'auto';
```

Comment:
There are a lot of instances of these repeated strings, so I did this so that we might be able to save a few bytes here and there with minification.